it/controllers/ChargeSummaryControllerISpec.scala

```diff
@@ -412,6 +412,240 @@ class ChargeSummaryControllerISpec extends ComponentSpecBase {
     }
   }
 
+  "MFADebits feature on Charge Summary Page" should {
+    val financialDetailsUnpaidMFA = Json.obj(
```

Comment:
These variables could potentially go in the FinancialDetailsIntegrationTestConstants